cypress/integration/main_page_spec.js

```diff
@@ -62,7 +66,11 @@ describe('Dashboard Home Page', () => {
       // cy.visit('/') should always work.
       //
       // https://github.com/cypress-io/cypress/issues/1311#issuecomment-393896371
+
+      // Logging to debug intermittent timeouts
+      cy.task('log', 'Logout');
```

Comment:
yeah so I think `cy.log` only prints to the cypress command log, whereas this would print it out to the terminal and you'd see it in the CI results, which helps debugging those CI only issues.